src/script/conversation/MessageRepository.ts

```diff
@@ -1197,7 +1218,7 @@ export class MessageRepository {
       );
     }
 
-    if (ensureUser && message.from && !message.user().id) {
+    if (message.from && !message.user().id) {
```

Comment:
Let's extract this logic to a different method and have a single responsibility for the `getMessageInConversationByReplacementId`. 
This way the consumer can do:

```js
const event = getMessageInConverstation(..) || getMessageInConversationByReplacementId()
const event = loadUsers(event);
```